components/button/BackBtn.tsx

```diff
@@ -0,0 +1,15 @@
+import "./Button.module.css";
+import styles from "./Button.module.css";
+
+function BackBtn() {
+  return (
+    <button className={`${styles.btn} ${styles.backBtn}`}>
+      <div className={styles.backContainer}>
+        <img src="/arrow.svg" alt="arrow" />
+        Back
+      </div>
+    </button>
```

Comment:
You can remove the `div` element.

```suggestion
    <button className={`${styles.btn} ${styles.backBtn} ${styles.backContainer}`}>
      <img src="/arrow.svg" alt="arrow" />
      Back
    </button>
```

You could make use of https://www.npmjs.com/package/classnames for easier `className` control.
See https://nextjs.org/learn/basics/assets-metadata-css/styling-tips